src/common.js

```diff
@@ -4,11 +4,23 @@
 
 var os = require('os');
 var fs = require('fs');
+var path = require('path');
 var glob = require('glob');
 var shell = require('..');
 
 var DEFAULT_ERROR_CODE = 1;
 var shellMethods = Object.create(shell);
+var PATH = 'PATH';
+
+// windows calls it's path 'Path' usually, but this is not guaranteed.
```

Comment:
Hmmm could you link me to the line in npm? I know that `process.env` is case-insensitive on my Windows machine. Perhaps it's not the case for everyone?

I would imagine things would get very broken in NodeJS if `process.env` were case-sensitive on Windows. Perhaps that's the case for older versions of Node?